packages/react/src/List/List.module.css

```diff
@@ -0,0 +1,35 @@
+.List {
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  list-style: none;
+  gap: var(--base-size-8);
+  padding-inline-start: 0px;
+  font-family: var(--brand-body-fontFamily);
+  font-weight: var(--brand-text-weight-400);
+  font-size: var(--brand-text-size-200);
+  line-height: var(--brand-text-lineHeight-400);
+  color: var(--brand-color-text-default);
+}
+
+ol.List {
+  counter-reset: li;
+}
+
+.ListItem {
+  display: flex;
+  gap: var(--base-size-12);
+  align-items: center;
+}
+
+.ListItem__leading-visual {
+  fill: var(--brand-color-text-muted);
+}
+
+ol.List > .ListItem::before {
```

Comment:
This is making me wonder if I forgot to enable stylelint 🙀, because linter should usually flag this as a violation. We don't use tag in selector to qualify the class, there's a specific rule around this. 

I'd approach this slightly differently. There are examples in other components too that might be worth checking out.

```css
.OrderedList__item::before {
```

OrderedList being the block, and item being the element of it. No need for qualifying selector then, or direct descendent. Can we do that?

Reply:
This doesn't quite work because you want the CSS split and the modules for ListItem and OrderedList don't have a relation to one another. This is the pro for the consolidated css.

Reply:
Pushed in changes